### PR TITLE
OD-96: add "Program" extra meta data field.

### PR DESCRIPTION
**Pull request by sivang**
Added "program" extra field.

### CODE REVIEW

File: ckanext/ed/schemas/dataset.json [hunk 1]
```diff
@@ -41,6 +41,20 @@
       "preset": "dataset_organization",
       "help_text": "The publishing entity and optionally their parent organization(s)."
     },
+    {
+      "field_name": "program",
+      "label": "Program Affiliation",
+      "preset" : "select",
+      "help_test": "The program this data profile is associated with. ",
```
Comment (zelima) [inline ckanext/ed/schemas/dataset.json]: Why trailing space at the end?

File: ckanext/ed/schemas/dataset.json [hunk 2]
```diff
@@ -41,6 +41,20 @@
...
+      "choices": [
+        {"value" : "housing",
```
Comment (zelima) [inline ckanext/ed/schemas/dataset.json]: Let's stick to the standards from the previous coder and keep code tidy and consistant:
```
# this looks better
{ 
  "value" :  "X",
  "label" : "Y"
}

# than this
{ "value" :  "X",
  "label" : "Y"
}

```

Comment (sivang) [reply ckanext/ed/schemas/dataset.json]: totally agreed! this was a pycharm auto formatting thing I missed.

File: ckanext/ed/schemas/dataset.json (hunk 1 above)
Comment (sivang) [reply ckanext/ed/schemas/dataset.json]: dropped

**Pull request merged**